system/include/GLFW/glfw3.h

```diff
@@ -1,9 +1,9 @@
 /*************************************************************************
- * GLFW 3.2 - www.glfw.org
```

Comment:
Can you mention PR description where this header file came from.  e.g. did you download it from github or from some source releases?

Reply:
aye.https://github.com/glfw/glfw/blob/master/include/GLFW/glfw3.h

Reply:
I wonder if its worth pulling it from a specific release? (and can you note that in the PR description).